docs/EN_US/HPCCSystemAdmin/SA-Mods/DAFtslave.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="SA_SupportUtilities">
+  <title>Support Utilities</title>
+
+  <para>There are a few additional components which are neither system servers
+  nor client interfaces but nonetheless important in supporting HPCC
+  tasks.</para>
+
+  <sect3 id="DAFilesrv">
+    <title>dafilesrv</title>
+
+    <para>The dafilesrv is a daemon that runs on system servers or physical
+    locations on the HPCC system. This daemon enables the HPCC components to
+    have file access to physical file locations. File access could be
+    spraying, despraying, copying, accessing logs, etc. Anything that requires
+    a file from another node, must have dafilesrv running on that node. The
+    dafilesrv daemon can be configured to allow SSL connections only, which
+    secures remote access to physical files.</para>
+
+    <para>The dafilesrv daemon runs as a process on every HPCC node. The
+    daemon will typically keep running even if your system is stopped. That is
+    an important fact to keep in mind if you are stopping your system for
+    maintenance, installations, or upgrades. Even if you issue the HPCC system
+    stop command, you would still need to stop the dafilesrv daemon. </para>
+
+    <para>One way to check and see if dafilesrv is running, is to issue a
+    command such as the following:</para>
+
+    <para><programlisting>ps -eaf | grep dafilesrv</programlisting> Even if
+    your HPCC System is stopped, you should still see that the dafilesrv is
+    running. </para>
+
+    <para>Issue the following command to stop the dafilesrv daemon. </para>
+
+    <programlisting>sudo /etc/HPCCSystems/sbin/hpcc-run.sh -a dafilesrv stop</programlisting>
```

Comment:
should probably also add the systemd alternative here?

Also should be OPT not ETC for the root of this.